decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -25,10 +25,11 @@ def apple_favicon
     end
 
     def legacy_favicon
-      icon_image = current_organization.attached_uploader(:favicon).variant_url(:small, host: current_organization.host)
+      variant = current_organization.favicon.content_type == "image/vnd.microsoft.icon" ? nil : :favicon
```

Comment:
I would do it in 2 lines to improve the readability, instead of an inverted if  (ex: `if true ? nil : value`)... 
```
variant = nil 
variant = :favicon unless current_organization.favicon.content_type == "image/vnd.microsoft.icon"
```

Reply:
Applied at f34104b94b7c1ae5150812ebc1ad8922b1be851d.